proto/beacon/p2p/v1/types.pb.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	_ "github.com/gogo/protobuf/gogoproto"
 	proto "github.com/gogo/protobuf/proto"
+	github_com_prysmaticlabs_go_bitfield "github.com/prysmaticlabs/go-bitfield"
```

Comment:
can we change the aliasing here ? its pretty ugly

Reply:
This is generated code. It's ok for it to be ugly